asyncstdlib/asynctools.py

```diff
@@ -174,3 +177,33 @@ async def head_tail(iterable, leading=5, trailing=5):
     if not hasattr(iterator, "aclose"):
         return nullcontext(iterator)
     return _ScopedAsyncIteratorContext(iterator)
+
+
+async def apply(
+    func: Callable[..., T], *args: Awaitable[Any], **kwargs: Awaitable[Any]
```

Comment:
Allow using ``func`` as a keyword.
```suggestion
    __func: Callable[..., T], *args: Awaitable[Any], **kwargs: Awaitable[Any]
```